### ISSUE TITLE
Cancel the EP400 once contentions have been migrated

### ISSUE THREAD

**Issue by amylai-nava** [department-of-veterans-affairs/abd-vro#2056]
Once #2054 is complete, the EP400 should be canceled.

## **Acceptance Criteria**
Within **EP Merge App (VRO)**:

Once the merge is complete (i.e., the contention was moved or confirmed to already be on the pending EP),
- Set the EP400's lifecycle status = Canceled
- Set the cancellation reason to:
  - `Issues moved into or confirmed in pending EP[x] - claim #[y]`
  - where [x] is the EP code and [y] is the benefit claim ID
- ~~Once the EP400 has been cancelled, the cancellation reason should be displayed as a claim note~~ - Moved to new ticket - (Derek)

**Comment by dfitchett** [comment]
list of reasons can be found by calling this [endpoint](https://claims-test.dev8.bip.va.gov/swagger-ui.html#/lookup/getLcStatusReasonTypes) (on VA network).

**Comment by amylai-nava** [comment]
@dfitchett thanks for linking the slack thread! I also updated the description of the ticket to reflect the new reason.
